src/components/map/Mapbox.vue

```diff
@@ -64,6 +111,14 @@ export default {
     navigateToEvent(event) {
       const { _id: id } = event;
       this.$router.push({ path: `/eventpage/${id}` });
+    },
+    updateQuery(payload) {
+      // console.log(payload);
+      const results = this.geocoderControl.query(payload);
+      // console.log(results);
+    },
+    handleSearch(event) {
+      // console.log(event);
```

Comment:
```suggestion
      // TODO: middle marker, and possible backend stuff for 'user suggestions'
```

The only use for handleSearch() I can think of is to put a temporary marker in the middle of where the user searched to indicate the exact location. I think this is a good idea - done this before on leaflet, should be pretty easy.

Reply:
Sure let's do that